### PR TITLE
Phone Environment Testing works (Install & Administrator)

### PR DESCRIPTION
**Pull request by gauravmittal1995**
The Install and Administrator Acceptance Tests work with phone environment

Depends on: https://github.com/joomla-projects/joomla-browser/pull/104

### CODE REVIEW

**Comment by gauravmittal1995** [comment]
@puneet0191  Hi, Can you please review this and let me know if i am proceeding in the correct direction?

**Comment by puneet0191** [comment]
@gauravmittal1995 With your changes, tests are failing on Travis

https://travis-ci.org/joomla-extensions/weblinks/jobs/114619924#L1956 See this
Can you think of a reason why it would fail?

**Comment by gauravmittal1995** [comment]
@puneet0191 Hi, It uses a new function in Joomla Browser which i have used. 
the PR is still open in joomla Browser for it.

PR: https://github.com/joomla-projects/joomla-browser/pull/104

**Comment by puneet0191** [comment]
Good :+1:

**Comment by puneet0191** [comment]
Prepare your Proposal, we will talk soon.

File: tests/acceptance/administrator/AdministratorWeblinksCest.php [hunk 1]
```diff
@@ -90,28 +95,32 @@ public function administratorTrashWeblink(AcceptanceTester $I)
...
+			$I->waitForText('Web Links','30',['css' => 'h1']);
+			$I->expectTo('see weblinks page');
+			$I->selectOptionInChosen('- Select Status -', 'Trashed');
+			$I->amGoingTo('Search the just saved weblink');
+			$I->searchForItem($this->title);
+			$I->waitForText('Web Links','30',['css' => 'h1']);
+
+			$I->amGoingTo('Delete the just saved weblink');
+			$I->checkAllResults();
+			$I->click(['xpath'=> '//div[@id="toolbar-delete"]/button']);
```
Comment (yvesh) [inline tests/acceptance/administrator/AdministratorWeblinksCest.php]: Does $I->clickToolbarButton('Delete'); not work?

**Comment by gauravmittal1995** [comment]
@yvesh Hi, I changed it to $I->clickToolBar('empty trash');

Can u please review??

**Comment by yvesh** [comment]
@gauravmittal1995 This PR has merge conflicts and need some improvements. Closing for now. If you want you can do a clean pull request again! Thank you for your contribution!

**Pull request closed without merging**